tools/code_format/check_format.py

```diff
@@ -111,7 +111,8 @@
 PROTO_VALIDATION_STRING = re.compile(r'\bmin_bytes\b')
 VERSION_HISTORY_NEW_LINE_REGEX = re.compile("\* ([a-z \-_]+): ([a-z:`]+)")
 VERSION_HISTORY_SECTION_NAME = re.compile("^[A-Z][A-Za-z ]*$")
-RELOADABLE_FLAG_REGEX = re.compile(".*(.)(envoy.reloadable_features.[^ ]*)\s.*")
+RELOADABLE_FLAG_REGEX = re.compile(".*(..)(envoy.reloadable_features.[^ ]*)\s.*")
+INVALID_REFLINK = re.compile(".* ref:.*")
```

Comment:
is this the only way the reflink is usually invalid? Or is this a way that sphinx compiles/only warns about? 

The reason I am asking is I wonder if we should check for this on formatting or if we should allow the sphinx compiler to complain

Reply:
It's totally legal.
Prior check mandated that reloadable flags were enclosed in single backticks, but on the latest release Asra caught that two flags out of something like 10 had double backticks, causing formatting inconsistencies by looking different.  This is just for visual consistency.